### PR TITLE
Feature: Adding Indiv Tutorials

### PR DESCRIPTION
**Pull request by darkvoid32**
Adding the individual tutorial content to the screens. I have hard-coded retrieving the content for each of the screens, since I'm not sure if theres a better, more scalable way to handle this. 

The title and headers are centered, and the paragraphs are aligned left.

### CODE REVIEW

**Comment by thunderbiscuit** [comment]
Yeah this is awesome! Just having the content there is already a big win, plus the fact that it's all handled in one file is just so much better than my previous 8-file version.

A few ideas:

1. Why don't we create an enum for the tutorials? I haven't thought through the exact way you'd have to make it work with the navigator which requires a string I think, but maybe something like:
```kotlin
enum class Tutorial(val id: String) {
    T0("0")
    T1("1")
    // ...
}
```
This would enable the removal of the `else` clause at the end of the `when` statement.

File: app/src/main/java/com/goldenraven/padawanwallet/ui/tutorials/TutorialsScreen.kt [hunk 1]
```diff
@@ -3,20 +3,141 @@ package com.goldenraven.padawanwallet.ui.tutorials
...
+import com.goldenraven.padawanwallet.R
 import com.goldenraven.padawanwallet.theme.md_theme_dark_background
 
 @Composable
-fun TutorialsScreen(navController: NavController, tutorialId: String) {
+fun TutorialsScreen(tutorialId: String) {
+
+    val title: String
+    val content: List<List<String>>
+    when (tutorialId.toInt() + 1) {
```
Comment (thunderbiscuit) [inline app/src/main/java/com/goldenraven/padawanwallet/ui/tutorials/TutorialsScreen.kt:25]: Your enum could go here.

File: app/src/main/java/com/goldenraven/padawanwallet/ui/tutorials/TutorialsScreen.kt [hunk 2]
```diff
@@ -3,20 +3,141 @@ package com.goldenraven.padawanwallet.ui.tutorials
...
+            )
+        }
+        8 -> {
+            title = stringResource(R.string.E8_title)
+            content = listOf(
+                listOf("h1", stringResource(R.string.t8_h1)),
+                listOf("p1", stringResource(R.string.t8_p1))
+            )
+        }
+        else -> {
```
Comment (thunderbiscuit) [inline app/src/main/java/com/goldenraven/padawanwallet/ui/tutorials/TutorialsScreen.kt:96]: Would allow to remove this `else` clause.

**Review by thunderbiscuit — COMMENTED** [review]
Let me know what you think of the enum idea for identifying tutorials

**Comment by thunderbiscuit** [comment]
FYI the keyword I try to use for new features in the commit message is `Feat`. So far I haven't used formal tools to search the codebase or triage commits and whatnot, but if ever we do it will be helpful to have all the same tags spelled the same.

**Comment by darkvoid32** [comment]
Added the enum and relevant code, though I think I could use some feedback on the way I went about it.

**Comment by thunderbiscuit** [comment]
Ok so my initial idea wasn't so good because I had not internalized that you really only pass strings to the navigation in Compose (I initially pictured just passing the enum for tutorials, which would have been so much simpler).

Here is an idea: make the `TutorialsScreen` take the `Tutorial` enum, and reconstruct that enum on the navigation side, before providing it as an argument to the Screen. See my commit for an implementation idea.

**Review by thunderbiscuit — APPROVED** [review]
ACK f63ff70af75a856a1935bc91c5cc354077b1eb8d. Good to go.

**Pull request merged**